docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,145 +1,102 @@
 # VIC Model Parameters
 
-!!! Warning
-    Docs not written yet.
+To convert ASCII parameter files that were used in previous versions of VIC, you may follow instructions laid out in [ASCII to NetCDF Parameter Conversion](Ascii_to_NetCDF_params.md).
 
-# Converting from VIC 4 to VIC 5
+We have also put together an example [Ipython notebook](../../../../samples/notebooks/example_reformat_vic4_parameters_to_vic5image.ipynb), that provides the steps necessary to do this conversion.
 
-We are developing a few scripts to aid users in moving from VIC 4 to VIC 5.  These scripts are currently distributed via the [`tonic`](https://github.com/UW-Hydro/tonic) [Python](https://www.python.org/) package.
+After going through these steps and setting up the tonic virtual environment, to look at the parameter file, you can use 'nco' to show the file contents:
 
-### Option 1: Installing Tonic Using Anaconda
-
-The easiest way to install Tonic and its dependencies is to use the [Anaconda Python distribution](https://store.continuum.io/cshop/anaconda/). The instructions below assume the user is using the `bash` shell.  `tonic` supports the use of Python 2.7 or 3.4+.
-
-To install Anaconda, follow these two simple steps (check to make sure the installer version is the most current)
-
-1.  download and run the Anaconda installer:  [http://continuum.io/downloads](http://continuum.io/downloads)
-
-2.  setup a virtual environment for Tonic
 ```shell
-conda create -n tonic python=3.4 netcdf4 pandas numpy scipy matplotlib xray basemap
-source activate tonic
+ncdump -h example.params.vic5.nc
```

Comment:
add example output of the ncdump.  At a minimum, people need to know what variables are in the parameter file and what format. They should not have to run tonic to figure that out.